README.win32

```diff
@@ -135,10 +135,8 @@ These free versions of Visual C++ 2013-2022 Professional contain the same
 compilers and linkers that ship with the full versions, and also contain
 everything necessary to build Perl.
 
-These packages can be downloaded by searching in the Download Center at
-L<https://www.microsoft.com/downloads/search.aspx?displaylang=en>.  (Providing exact
-links to these packages has proven a pointless task because the links keep on
-changing so often.)
+These packages can be downloaded by following links to executables at
+https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist
```

Comment:
Probably need to be enclosed within `L<>`

Reply:
Amended to correct that updating commit title as well (added `perlwin32:` prefix for a single file change to match PR's).